scripts/cleanup_leftover_test_orgs.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+if [[ "$(cf api | head -n1 | awk '{print $3}')" =~ ^.*cloud.service.gov.uk$ ]]; then
+  echo "Don't try to use this script in prod. It's a bit too risky."
```

Comment:
Actually given that there is later confirmation, I disagree

Reply:
Fair point!